cdk/lib/utils.py

```diff
@@ -45,6 +45,7 @@ def get_private_client_ids(config_dir: str) -> list[dict[str, str]]:
                                     {
                                         "id": client["clientId"],
                                         "realm": data.get("realm", ""),
+                                        "application_role_arn": client.get("application_role_arn", ""),
```

Comment:
Recommend going with `None` rather than an empty string to really evoke that it wasn't set on the upstream config.

```suggestion
                                        "application_role_arn": client.get("application_role_arn"),
```